tests/hypertext/htmlhint_test_files/.htmlhintrc

```diff
@@ -0,0 +1,4 @@
+{
+    "inline-script-disabled": true,
+    "inline-style-disabled": true
+}
```

Comment:
I've faced this issue a lot of times. Should I open an issue for a `NewlineBear` or similar ?